MAUI/PDF-Viewer/Signature.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Electronic Signature in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about Electronic Signature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and its types.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Electronic Signature in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in PDF Viewer for working with signature.
```

Comment:
The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in the PDF Viewer for working with signature.